curl.ml

```diff
@@ -494,6 +494,9 @@ type curlOption =
   | CURLOPT_XFERINFOFUNCTION of (int64 -> int64 -> int64 -> int64 -> bool)
   | CURLOPT_PREREQFUNCTION of (string -> string -> int -> int -> bool)
   | CURLOPT_AWS_SIGV4 of string
+  | CURLOPT_TCP_KEEPALIVE of bool
+  | CURLOPT_TCP_KEEPIDLE of int64
```

Comment:
int64 is not convenient, int is enough

Reply:
thanks! addressed in https://github.com/ygrek/ocurl/pull/70/commits/d9bdf57bac1b08eeec278772489c6fad620f66dd